sway/input/seat.c

```diff
@@ -329,7 +329,7 @@ static struct sway_seat_node *seat_node_from_node(
 	seat_node->node = node;
 	seat_node->seat = seat;
 	wl_list_insert(seat->focus_stack.prev, &seat_node->link);
-	wl_signal_add(&node->events.destroy, &seat_node->destroy);
```

Comment:
I intended to separate out the refocus logic from destroy so it could be figured out later, but on closer inspection, it seems that after this change the node::destroy event is just unused, because this was actually its only listener.

So maybe there is no need for a new event after all. Is it still worth renaming it to "postdestroy" or should I just move the *emits around?